src/stories/components/svg/EndgameAtlasBudgets.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
```

Comment:
Why not add these icons to the component-scoped folder of the pages? As they are going to be used on the new page only, there would be a better place to live, if it is needed somewhere in the app later we can move it to the global folder later